src/Modules/OrchardCore.Commerce/Recipes/OrchardCore.Commerce.Development.Setup.recipe.json

```diff
@@ -88,6 +88,38 @@
         }
       ]
     },
+    {
+      "name": "Layers",
+      "Layers": [
+        {
+          "Name": "Always",
+          "Rule": null,
+          "Description": null,
+          "LayerRule": {
+            "Conditions": [
+              {
+                "$type": "OrchardCore.Rules.Models.BooleanCondition, OrchardCore.Rules",
+                "Value": true,
+                "Name": "BooleanCondition",
+                "ConditionId": "45njstmdjvxrv11m0zb80mjtw0"
+              }
+            ],
+            "Name": null,
+            "ConditionId": "4dt7eswwkkdjh5aryba5z86vws"
```

Comment:
What are these two condition IDs referring to? I don't see them anywhere else. If they are just arbitrary unique IDs, then use `[js:uuid()]`.